### PR TITLE
Add possibility to translate namespaced routes or segments

### PR DESCRIPTION
**Pull request by WhiteWolf99**
Example:

```php
Route::localized(function () {

    Route::get(Lang::uri('about/us', null, 'blog'), AboutController::class.'@index')
        ->name('about.us');

});
```

### CODE REVIEW

File: src/Macros/UriTranslationMacro.php [hunk 1]
```diff
@@ -28,11 +28,11 @@ public static function register()
             // Attempt to translate each segment. If there is no translation
             // for a specific segment, then its original value will be used.
             $translations = $segments->map(function ($segment) use ($locale) {
```
Comment (ivanvermeyen) [inline src/Macros/UriTranslationMacro.php:30]: Forgot to use $namespace here (see failing tests)

**Review by ivanvermeyen — CHANGES_REQUESTED** [review]
Thanks for the PR!
Some tests were failing, see comments in the PR code.

File: src/Macros/UriTranslationMacro.php (hunk 1 above)
Comment (WhiteWolf99) [reply src/Macros/UriTranslationMacro.php]: Thanks for the correction! Indeed, now it should pass all tests.

**Review by ivanvermeyen — APPROVED** [review]

**Comment by ivanvermeyen** [comment]
Thanks!

**Pull request merged**